tools/release/lp_build_monitor_recipe.py

```diff
@@ -47,7 +55,30 @@ def wait_every_build_started(build_recipe):
         pending_builds = build_recipe.getPendingBuildInfo()
 
 
-def get_new_builds(build_recipe, started_datetime):
+def recipe_name_to_source_name(name):
+    # name is in the form of source-name-with-dashes-risk
```

Comment:
are doctests included in the coverage reports? I think they are.
This is a great example how doctest here would be better than this explanation + navigating (searching for) tests.

Also why not a docstring?

Reply:
They are not executed I think, I have written a normal test for this. I will add the docstring.

Reply:
Yeah, I saw the test (tbh, this is how I fully understood what it does :D